src/__swaps__/utils/numbers.ts

```diff
@@ -270,28 +268,65 @@ export const convertBipsToPercentage = (value: BigNumberish, decimals = 2): stri
 /**
  * @desc convert from amount value to display formatted string
  */
-export const convertAmountToNativeDisplay = (
-  value: BigNumberish,
-  nativeCurrency: keyof nativeCurrencyType,
-  buffer?: number,
-  skipDecimals?: boolean
-) => {
+export const convertAmountToNativeDisplay = (value: number | string, nativeCurrency: keyof nativeCurrencyType, useThreshold = false) => {
   const nativeSelected = supportedNativeCurrencies?.[nativeCurrency];
-  const { decimals } = nativeSelected;
-  const display = handleSignificantDecimals(value, decimals, buffer, skipDecimals);
-  if (nativeSelected.alignment === 'left') {
-    return `${nativeSelected.symbol}${display}`;
+  const { alignment, decimals: rawDecimals, symbol } = nativeSelected;
+  const decimals = Math.min(rawDecimals, 6);
```

Comment:
my only issue with this is when the native currency is the added zeroes when not needed
<img width="559" alt="Screenshot 2024-06-12 at 11 36 44 AM" src="https://github.com/rainbow-me/rainbow/assets/6284525/9b4a42d3-e1ca-4e32-8890-bf26a181d1cd">